### ISSUE TITLE
Updates for version 0.11

### ISSUE THREAD

**Issue by suzumiyasmith** [slamdata/purescript-markdown#78]
Error 6 of 8:

  at bower_components/purescript-markdown/src/Text/Markdown/SlamDown/Eval.purs line 35, column 1 - line 35, column 1

    Unable to parse module:
    unexpected comma in constraints.

Class constraints in type annotations can no longer be grouped in parentheses.
Each constraint should now be separated by `=>`, for example:
    `(Applicative f, Semigroup a) => a -> f a -> f a`
  would now be written as:
    `Applicative f => Semigroup a => a -> f a -> f a`.

    expecting indentation past column 1, (, qualifier, proper name, operator or ->

  See https://github.com/purescript/documentation/blob/master/errors/ErrorParsingModule.md for more information,
  or to contribute content related to this error.

Error 7 of 8:

  at bower_components/purescript-markdown/src/Text/Markdown/SlamDown/Traverse.purs line 99, column 1 - line 99, column 1

    Unable to parse module:
    unexpected comma in constraints.

Class constraints in type annotations can no longer be grouped in parentheses.
Each constraint should now be separated by `=>`, for example:
    `(Applicative f, Semigroup a) => a -> f a -> f a`
  would now be written as:
    `Applicative f => Semigroup a => a -> f a -> f a`.

    expecting indentation past column 1, (, qualifier, proper name, operator or ->

  See https://github.com/purescript/documentation/blob/master/errors/ErrorParsingModule.md for more information,
  or to contribute content related to this error.

Error 8 of 8:

  at bower_components/purescript-markdown/src/Text/Markdown/SlamDown/Syntax/FormField.purs line 265, column 1 - line 265, column 1

    Unable to parse module:
    unexpected comma in constraints.

Class constraints in type annotations can no longer be grouped in parentheses.
Each constraint should now be separated by `=>`, for example:
    `(Applicative f, Semigroup a) => a -> f a -> f a`
  would now be written as:
    `Applicative f => Semigroup a => a -> f a -> f a`.

    expecting indentation past column 1, (, qualifier, proper name, operator or ->

  See https://github.com/purescript/documentation/blob/master/errors/ErrorParsingModule.md for more information,
  or to contribute content related to this error.

* ERROR: Subcommand terminated with exit code 1

**Comment by suzumiyasmith** [comment]
I've make [some changes](https://github.com/suzumiyasmith/purescript-markdown/commits/master), and fix some dependencies and grammar issue.
But I stuck with the error below.
```
* Building project in /home/suzumiya/Code/purescript-markdown
Error found:
in module Text.Markdown.SlamDown.Syntax.Inline
at src/Text/Markdown/SlamDown/Syntax/Inline.purs line 58, column 1 - line 58, column 51

  No type class instance was found for

    Data.Ord.Ord1 Expr


while applying a function compare
  of type Ord t0 => t0 -> t0 -> Ordering
  to argument $l57
while inferring the type of compare $l57
in value declaration ordInline

where t0 is an unknown type

See https://github.com/purescript/documentation/blob/master/errors/NoInstanceFound.md for more information,
or to contribute content related to this error.


Compiling Text.Markdown.SlamDown.Syntax.Inline
* ERROR: Subcommand terminated with exit code 1

```

**Comment by kRITZCREEK** [comment]
Fixed by #79
